pkg/services/folder/model.go

```diff
@@ -16,6 +16,7 @@ var ErrDatabaseError = errutil.Internal("folder.database-error")
 var ErrInternal = errutil.Internal("folder.internal")
 var ErrCircularReference = errutil.BadRequest("folder.circular-reference", errutil.WithPublicMessage("Circular reference detected"))
 var ErrTargetRegistrySrvConflict = errutil.Internal("folder.target-registry-srv-conflict")
+var ErrFolderContainsAlertRules = errutil.BadRequest("folder.contains-alert-rules", errutil.WithPublicMessage("Folder cannot be deleted: folder contains alert rules"))
 
```

Comment:
I wonder if it would be better to have msgID more generic "folder.non-empty". It's simpler to change the public message than msgId.

Reply:
fixed in  a869362708c66f935fb8f7cfcd925cb255d61b7f